charts/apisix/templates/service-gateway.yaml

```diff
@@ -76,6 +76,24 @@ spec:
     port: {{ .port }}
     targetPort: {{ .port }}
   {{- end }}
+  {{- if or .Values.service.proxyProtocol.http.enabled }}
+  - name: apisix-gateway-pp-http
+    port: {{ .Values.service.proxyProtocol.http.servicePort }}
+    targetPort: {{ .Values.service.proxyProtocol.http.containerPort }}
```

Comment:
I realize you didn't define container port in `templates/deployment.yaml`. The request sent to the proxy_protocol port can't be forwarded to Apisix. Please explain to me if I am wrong.